examples/unstructured_2d_dgsem/elixir_euler_basic.jl

```diff
@@ -58,11 +58,15 @@ save_solution = SaveSolutionCallback(interval = 10,
 
 stepsize_callback = StepsizeCallback(cfl = 0.9)
 
+time_series = TimeSeriesCallback(semi, [(2.0, -0.5), (0.28, 1 / 3), (1.87, 2 / 3)];
+                                 interval = 10)
```

Comment:
Maybe add a test for the resulting file in the test setup where this elixir is used?

Reply:
Good idea, I can add this once I change which elixir will contain the `TimeSeriesCallback`.

Reply:
I have added such a test.

Reply:
Thanks!